go/merge/three_way_keyval_test.go

```diff
@@ -64,18 +72,22 @@ var (
 
 type ThreeWayMergeSuite struct {
```

Comment:
Sure

On Thu, Aug 25, 2016 at 1:59 PM Ben Kalman <notifications@github.com> wrote:

> In go/merge/three_way_keyval_test.go
> <https://github.com/attic-labs/noms/pull/2438#discussion_r76323315>:
>
> > @@ -64,18 +72,22 @@ var (
> >
> >  type ThreeWayMergeSuite struct {
>
> Sorry... could you leave the stuff that applies to both set and keyval
> tests in a three_way_test.go file? I.e. ThreeWayMergeSuite and friends.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/attic-labs/noms/pull/2438/files/3f7c8f00f241263997213f718a0cfb9ee816d84b#r76323315>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMnImqgyHfLc3qiRAbSkfmamRktS2aNAks5qjgIqgaJpZM4JtfQm>
> .
>